app/src/app/components/contents/metadata/Metadata.tsx

```diff
@@ -3,57 +3,107 @@ import { getDateUTCString, getElapsedTime } from "../../../../helpers";
-                      <ul className="ps-1 list-disc list-inside">
-                        {packet.git.url?.map((url, index) => (
-                          <li key={index} className="text-muted-foreground">
-                            {url}
-                          </li>
-                        ))}
-                      </ul>
-                  </li>
+              </AccordionTrigger>
+              <AccordionContent>
+                <ul className="ps-1 flex gap-10">
+                  {startedTime && <MetadataListItem label="Started" value={startedTime} />}
+                  {elapsedTime && <MetadataListItem label="Elapsed" value={elapsedTime} />}
                 </ul>
-              </div>
-            </>
-          )}
+              </AccordionContent>
+            </AccordionItem>
+            {packet.git && (
+              <>
+                <AccordionItem value="git">
+                  <AccordionTrigger>
+                    <span className="flex gap-1 items-center">
+                      <Github className="small-icon text-muted-foreground" />
+                      <h3 data-testid="gitHeading">Git</h3>
+                    </span>
+                  </AccordionTrigger>
+                  <AccordionContent>
+                    <ul className="space-y-1">
+                      <MetadataListItem label="Branch" value={packet.git.branch} />
+                      <MetadataListItem label="Commit" value={packet.git.sha} />
+                      <li className="flex flex-col">
+                        <span className="font-semibold mr-2">Remotes</span>
+                        <ul className="ps-1 list-disc list-inside">
+                          {packet.git.url?.map((url, index) => (
+                            <li key={index} className="text-muted-foreground">
+                              {url}
+                            </li>
+                          ))}
+                        </ul>
+                      </li>
+                    </ul>
+                  </AccordionContent>
+                </AccordionItem>
+              </>
+            )}
+            {session && (
+              <>
+                <AccordionItem value="platform">
+                  <AccordionTrigger>
+                    <span className="flex gap-1 items-center">
+                      <Monitor className="small-icon text-muted-foreground" />
+                      <h3>Platform</h3>
+                    </span>
+                  </AccordionTrigger>
+                  <AccordionContent>
+                    <ul className="space-y-1">
+                      <MetadataListItem label="OS" value={session.platform.os} />
+                      <MetadataListItem label="System" value={session.platform.system} />
+                      <MetadataListItem label="Language" value={session.platform.version} />
+                    </ul>
+                  </AccordionContent>
+                </AccordionItem>
+                <AccordionItem value="packages">
```

Comment:
Does this want to be conditional on `packages` being truthy? There's a `session?.packages?` above which implies packages may not be set even when sessions is. But I'm not sure if this ever happens!

Reply:
Oh I see!